core/src/storage2/collections/binary_heap/tests.rs

```diff
@@ -327,13 +354,21 @@ fn push_largest_value_complexity_big_o_log_n() -> env::Result<()> {
 #[test]
 fn push_smallest_value_complexity_big_o_1() -> env::Result<()> {
     const SMALLEST_VALUE: u32 = 0;
-    const EXPECTED_READS: usize = 3;
+
+    // 1 wrapper overhead (#508) + 1 wrapper.len + 1 vec overhead (#508) +
+    // 1 vec.len + 1 vec.cell in which to insert + 1 parent cell during `sift_up`
+    const EXPECTED_READS: usize = 6;
```

Comment:
The constant `O(1)` is doubling here, which is fine but the cost of this perhaps needs to be factored in to the improvement we get in `pop` from the added overhead